### PR TITLE
Refs #9099 - Update rubocop to 0.28.0

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: test/controllers/api/v2/sync_plans_controller_test.rb [hunk 1]
```diff
@@ -168,13 +168,13 @@ def test_add_products_protected
...
         put :add_products, :id => @sync_plan.id, :organization_id => @organization.id,
-          :product_ids => @products.collect { |p| p.id }
+                           :product_ids => @products.collect { |p| p.id }
       end
     end
 
     def test_remove_products
       put :remove_products, :id => @sync_plan.id, :organization_id => @organization.id,
-          :product_ids => @products.collect { |p| p.id }
+                            :product_ids => @products.collect { |p| p.id }
```
Comment (mccun934) [inline test/controllers/api/v2/sync_plans_controller_test.rb]: are these style indents required? is it required to line up with the second arg in the call?

File: test/lib/util/package_clause_generator_test.rb [hunk 2]
```diff
@@ -61,9 +61,9 @@ def test_package_names
     def test_package_versions
       @filter = FactoryGirl.create(:katello_content_view_package_filter, :content_view => @content_view)
       foo_rule = FactoryGirl.create(:katello_content_view_package_filter_rule, :filter => @filter,
-                                    :name => "foo*", :version => "5.0")
+                                                                               :name => "foo*", :version => "5.0")
       goo_rule = FactoryGirl.create(:katello_content_view_package_filter_rule, :filter => @filter,
-                                    :name => "goo*", :min_version => "0.5", :max_version => "0.7")
+                                                                               :name => "goo*", :min_version => "0.5", :max_version => "0.7")
```
Comment (mccun934) [inline test/lib/util/package_clause_generator_test.rb]: another strange looking indent

File: test/controllers/api/v2/sync_plans_controller_test.rb (hunk 1 above)
Comment (daviddavis) [inline test/controllers/api/v2/sync_plans_controller_test.rb]: Yea, it's part of the hash that starts with `:id` so it should be lined up. That said, I don't have a strong preference for this rule though if we'd rather line up method arguments instead.

File: test/lib/util/package_clause_generator_test.rb (hunk 2 above)
Comment (daviddavis) [inline test/lib/util/package_clause_generator_test.rb]: Same rule/situation here.

Comment (mccun934) [inline test/lib/util/package_clause_generator_test.rb]: IMO, :-1: on that rule but I wouldn't block the PR on it

**Comment by jlsherrill** [comment]
ACK from me

**Comment by daviddavis** [comment]
Merging to fix master.

**Pull request merged**